app/src/main/kotlin/org/wordpress/aztec/demo/MainActivity.kt

```diff
@@ -180,39 +180,11 @@ open class MainActivity : AppCompatActivity(),
         private val MARK = "<p>Donec ipsum dolor, <mark style=\"color:#ff0000\">tempor sed</mark> bibendum <mark style=\"color:#1100ff\">vita</mark>.</p>"
 
         private val EXAMPLE =
-                IMG +
-                        HEADING +
-                        BOLD +
-                        ITALIC +
-                        UNDERLINE +
-                        STRIKETHROUGH +
-                        ORDERED +
-                        ORDERED_WITH_START +
-                        ORDERED_REVERSED +
-                        ORDERED_REVERSED_WITH_START +
-                        ORDERED_REVERSED_NEGATIVE_WITH_START +
-                        ORDERED_REVERSED_WITH_START_IDENT +
-                        LINE +
-                        UNORDERED +
-                        QUOTE +
-                        PREFORMAT +
-                        LINK +
-                        HIDDEN +
-                        COMMENT +
-                        COMMENT_MORE +
-                        COMMENT_PAGE +
-                        CODE +
-                        UNKNOWN +
-                        EMOJI +
-                        NON_LATIN_TEXT +
-                        LONG_TEXT +
-                        VIDEO +
-                        VIDEOPRESS +
-                        VIDEOPRESS_2 +
-                        AUDIO +
-                        GUTENBERG_CODE_BLOCK +
-                        QUOTE_RTL +
-                        MARK
+                """
+                    <br>
+                    <placeholder id="123" type="image_with_caption" src="https://filesamples.com/samples/image/jpg/sample_640%C3%97426.jpg" caption="caption of image"/>
+                    <br>
+                """.trimIndent()
```

Comment:
np: Probably don't want to commit this change since it removes the example blocks. Maybe instead add the placeholder as a sample block?

Reply:
I'll revert the `MainActivity` changes 👍